js/classes/entities/background/decoration.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * General-purpose class for displaying entites that do not move and which cannot be collided with.
+ * 
+ * @author Trae Claar
+ */
+class Decoration {
+
+    /**
+     * Constructor for a Decoration. The Decoration is positioned with respect to the BOTTOM (not top) left 
+     * corner.
+     * 
+     * @param {string} type the type of Decoration to construct (must be a key to Decoration.PROPERTY_TABLE)
+     * @param {Vector} pos the position of the bottom left corner of the Decoration (in world space) 
```

Comment:
I think, with a special position such as this, "pos" is a bit vague (although yes, the JSDoc does explain it). Consider bottomPos, posBL, something like that.

Reply:
Agreed. Good idea.

Reply:
Agreed, good suggestion. Thanks!